build/template-restore-build-libsandsamples.yaml

```diff
@@ -12,6 +12,21 @@ steps:
   inputs:
     script: 'nuget locals all -clear'
 
+#Restore workload with the .NET Core CLI task
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet workload restore'
```

Comment:
display name must be unique to identify the task it is performing, clashes with the below one